src/main/webapp/app/config/constants/regex.ts

```diff
@@ -9,3 +9,5 @@ export const UUID_REGEX = new RegExp('\\w{8}-\\w{4}-\\w{4}-\\w{4}-\\w{12}');
 export const WHOLE_NUMBER_REGEX = new RegExp('^\\d+$');
 
 export const INTEGER_REGEX = /^-?\d+$/;
+
+export const EXON_ALTERATION_REGEX = /^(Any\s+)?(Exon\s+(\d+)(-(\d+))?\s+(Deletion|Insertion|Duplication))(\s*\+\s*(\1))*/i;
```

Comment:
I was trying to use `\1` as a backreference to the group `(Exon\s+(\d+)(-(\d+))?\s+(Deletion|Insertion|Duplication))`, so that we can match exons that are seperated by a plus sign: ie `Exon 4 Deletion + Exon 6-8 Deletion + Exon 10 Deletion`.

I ended up changing the back reference to the regex format instead.